### PR TITLE
Update contact part in UG (excluding archive/unarchive related features)

### PR DESCRIPTION
**Pull request by Persdre**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -293,12 +293,12 @@ Action | Format, Examples
...
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMERGENCY_CONTACT [t/TAG]…` <br> e.g., `add n/James Ho p/22224444 e/81234567 t/morning shift`
-**List** | `list`
-**Edit** | `edit INDEX [n/NAME] [e/EMERGENCY_CONTACT] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/81234567`
-**Search** | `search KEYWORD [MORE_KEYWORDS]`<br> e.g., `search James Jake`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Clear** | `clear`
+**Add** | `c-add n/NAME p/PHONE_NUMBER e/EMERGENCY_CONTACT [t/TAG]…` <br> e.g., `c-add n/James Ho p/22224444 e/81234567 t/morning shift`
+**List** | `c-list`
+**Edit** | `c-edit INDEX [n/NAME] [e/EMERGENCY_CONTACT] [t/TAG]…​`<br> e.g.,`c-edit 2 n/James Lee e/81234567`
+**Search** | `c-find KEYWORD [MORE_KEYWORDS]`<br> e.g., `c-find James Jake`
```
Comment (yanbingtao) [inline docs/UserGuide.md]: **Search** should change to **Find**

**Review by ureshiiYing — DISMISSED** [review]
LGTM

**Review by yanbingtao — APPROVED** [review]
LGTM

**Review by Diwu-Yi — APPROVED** [review]
LGTM

**Pull request merged**